src/search/screens/search.screen.js

```diff
@@ -127,19 +161,22 @@ class Search extends Component {
     const selectedSearchType =
       selectedType !== null ? selectedType : this.state.searchType;
 
-    if (query !== '') {
+    if (query.trim() !== '') {
+      const searchedQuery = query.toLowerCase();
+
       this.setState({
         searchStart: true,
         currentQuery: {
           ...this.state.currentQuery,
-          [selectedSearchType]: query,
+          [selectedSearchType]: searchedQuery,
         },
-        query,
+        query: searchedQuery,
       });
-      if (selectedSearchType === 0) {
-        searchRepos(query);
+      this.props.navigation.setParams({ [NAV_QUERY_PARAM]: searchedQuery });
```

Comment:
We save the query string in the navigation, to be able to get it in MapStateToProps and select the corresponding pagination